src/Microsoft.Restier.Core/ApiConfiguration.cs

```diff
@@ -35,10 +39,8 @@ namespace Microsoft.Restier.Core
     /// </remarks>
     public class ApiConfiguration : PropertyBag
     {
-        private static ConcurrentDictionary<Type, Action<IServiceCollection>> configurations =
-            new ConcurrentDictionary<Type, Action<IServiceCollection>>();
-
-        private static Action<IServiceCollection> emptyConfig = _ => { };
+        private static ConcurrentDictionary<Type, ApiBuilder> configurations =
```

Comment:
ApiConfiguration maintains a global type to ApiBuilder map, but ApiBuilder could also be used standalone. And then build standalone ApiConfiguration.